types/bun/tty.d.ts

```diff
@@ -0,0 +1,18 @@
+
+declare module 'tty' {
```

Comment:
this one isn't actually implemented

we should do it, but let's not put it in types until it works